helpers/crud/cuentas_contables/comprobante.go

```diff
@@ -1,29 +1,50 @@
 package cuentas_contables
 
 import (
+	"encoding/json"
 	"fmt"
 	"net/http"
 	"strconv"
 
 	"github.com/astaxie/beego"
 	"github.com/astaxie/beego/logs"
+	"github.com/udistrital/movimientos_contables_mid/helpers"
 	e "github.com/udistrital/utils_oas/errorctrl"
 	r "github.com/udistrital/utils_oas/request"
 )
 
 func GetComprobanteById(id string, comprobante interface{}) (outputError map[string]interface{}) {
 	const funcion string = "GetComprobanteById"
+	var fullResponse map[string]interface{}
 	defer e.ErrorControlFunction(funcion+" - Unhandled Error!", strconv.Itoa(http.StatusInternalServerError))
 
 	url := beego.AppConfig.String("CuentasContablesCrudService") + "/comprobante/" + id
-	if resp, err := r.GetJsonTest(url, &comprobante); err != nil || resp.StatusCode != http.StatusOK {
+	if resp, err := r.GetJsonTest(url, &fullResponse); err != nil || resp.StatusCode != http.StatusOK {
 		status := http.StatusBadGateway
 		if err == nil { // resp.StatusCode != http.StatusOK
 			err = fmt.Errorf("undesired status code - %s", http.StatusText(resp.StatusCode))
 			status = resp.StatusCode
 		}
 		logs.Error(err)
-		outputError = e.Error(funcion+" - r.GetJsonTest(url, &comprobante)", err, strconv.Itoa(status))
+		outputError = e.Error(funcion+" - r.GetJsonTest(url, &fullResponse)", err, strconv.Itoa(status))
 	}
+	helpers.LimpiezaRespuestaRefactorBody(fullResponse, &comprobante)
 	return
 }
+
+func GetComprobanteWorker(etiquetaString string, c chan interface{}) {
+	var etiqueta map[string]string
+	if err := json.Unmarshal([]byte(etiquetaString), &etiqueta); err == nil {
+		id := fmt.Sprintf("%v", etiqueta["ComprobanteId"])
+		var comprobante interface{}
+		outputError := GetComprobanteById(id, &comprobante)
+		if outputError != nil {
+			c <- nil
```

Comment:
En caso de error y antes de "retornar"...
```suggestion
			logs.Warn(outputError)
			c <- nil
```
... mostrar el error en los logs, podría ser a nivel de `Warn`, `Error` u otro nivel según sea adecuado

(No aplicar este _suggested change_ directamente, ajustar según sea pertinente)